packages/client/types/modules.d.ts

```diff
@@ -19,6 +19,7 @@ declare module 'emoji-mart/dist-modern/utils/data.js'
 declare module 'emoji-mart/dist-modern/components/picker/nimble-picker'
 declare module 'react-textarea-autosize'
 declare module 'react-copy-to-clipboard'
+declare module 'react-dom/client'
```

Comment:
not sure why I'm getting even though the types were updated to the latest version
```
Could not find a declaration file for module 'react-dom/client'. '/Users/bjarocki/Work/parabol/web-parabol/node_modules/react-dom/client.js' implicitly has an 'any' type.
  If the 'react-dom' package actually exposes this module, consider sending a pull request to amend 'https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/react-dom'ts(7016)
```

Reply:
oh, now I know. version was locked in our root package.json resolutions!